### PR TITLE
{chem}[intel/2018b] Wannier90 v3.0.0

### PR DESCRIPTION
**Pull request by jmbeuken**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name = 'Wannier90'
+version = '3.0.0'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/wannier-developers/wannier90/archive/']
```
[the remark this answers is not in the document]
Comment (jmbeuken) [reply easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb]: ok...more stylish ;-)

File: easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+sources = ['v%(version)s.tar.gz']
+patches = ['Wannier90_3x_ignore_makeinc.patch']
+checksums = [
+    'f196e441dcd7b67159a1d09d2d7de2893b011a9f03aab6b30c4703ecbf20fe5b',   # v3.0.0.tar.gz
+    '561c0d296e0e30b8bb303702cd6e41ded54c153d9b9e6cd9cab73858e5e2945e',   # Wannier90_3x_ignore_makeinc.patch
+]
+
+buildopts = 'all F90=$F90 MPIF90=$MPIF90 FCOPTS="$FFLAGS" LDOPTS="$FFLAGS" '
+buildopts += 'LIBDIR="$MKLROOT/lib/intel64" '
```
[the remark this answers is not in the document]
Comment (jmbeuken) [reply easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb]: ok... done

File: easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+buildopts += 'LIBS="$LIBDIR -lmkl_core -lmkl_intel_lp64 -lmkl_sequential -lpthread" '
```
[the remark this answers is not in the document]
Comment (jmbeuken) [reply easybuild/easyconfigs/w/Wannier90/Wannier90-3.0.0-intel-2018b.eb]: done

File: easybuild/easyconfigs/w/Wannier90/Wannier90_3x_ignore_makeinc.patch [hunk 4]
```diff
@@ -0,0 +1,32 @@
+avoid including make.inc, which contains hardcoding settings we don't need/want
```
[the remark this answers is not in the document]
Comment (jmbeuken) [reply easybuild/easyconfigs/w/Wannier90/Wannier90_3x_ignore_makeinc.patch]: without the patch, the compilation failed with : 

```
make[1]: Entering directory `/home/users/j/m/jmbeuken/.local/easybuild/build/Wannier90/3.0.0/intel-2018b/wannier90-3.0.0/src/obj'
../Makefile.2:5: ../../make.inc: No such file or directory
make[1]: Entering directory `/home/users/j/m/jmbeuken/.local/easybuild/build/Wannier90/3.0.0/intel-2018b/wannier90-3.0.0/src/objp'
../Makefile.2:5: ../../make.inc: No such file or directory
make[1]: *** make[1]: *** No rule to make target `../../make.inc'No rule to make target `../../make.inc'.  Stop.
```

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/bc3973efadc72cc8dcf6cbe3ec8a96a3 for a full test report.

File: easybuild/easyconfigs/w/Wannier90/Wannier90_3x_ignore_makeinc.patch (hunk 4 above)
[the remark this answers is not in the document]
Comment (migueldiascosta) [reply easybuild/easyconfigs/w/Wannier90/Wannier90_3x_ignore_makeinc.patch]: sorry for not being clearer, I meant in comparison with the existing easyconfigs where one of the `config/make.inc.*` files is used, but that does make the easyconfig look toolchain dependent, when it isn't, so using your patch may be better

**Comment by migueldiascosta** [comment]
Going in, thanks @jmbeuken!

**Pull request merged**